app/views/layouts/_application.html.erb

```diff
@@ -1,6 +1,12 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
+    <% if Rails.env.to_s.in? Sentry.configuration.enabled_environments %>
+      <%= tag :meta, name: :release, content: Sentry.configuration.release %>
+      <%= tag :meta, name: :"sentry-dsn", content: Sentry.configuration.dsn.to_s %>
+      <%= javascript_pack_tag('sentry') unless params[:nojs] === "nojs" %>
```

Comment:
two equal signs please, we are still writing Ruby code here :)

Reply:
Good catch, I'll correct this in the place I've copied it from as well :P